### PR TITLE
Phenogrid fix

### LINKED ISSUE #443: ReferenceError: Phenogrid is not defined on analyze/phenotypes
On https://monarchinitiative.org/analyze/phenotypes

1. Select No
2. Generate from a gene
3. Type 'SHH'
4. Select Shh
5. Select 'Confirm Profile'
6. Select 'Show Me Everything'
7. Select 'All Matching Genes'
8. Click select by Taxon -> Mouse(genes)
9. Click run similarity analysis

I can create a gif since it's a lot of steps.

Seems related to the Phenogrid module being global instead of explicitly imported into https://github.com/monarch-initiative/monarch-ui/blob/63677/src/components/PhenoGrid.vue , but I can't reason about how this originally worked.

### PR DESCRIPTION
**Pull request by vincerubinetti**
closes #443 

@falquaddoomi fixed the upstream `phenogrid` issues in https://github.com/monarch-initiative/phenogrid/pull/282 . This updates the version of the `phenogrid` package being used and fixes the import. After the Monarch folks have tested this out in the deploy preview below, we can merge the `phenogrid` pr, publish the new version to npm, change this pr to install from npm (instead of the hacky install from @falquaddoomi 's github repo), then merge this.

### CODE REVIEW

**Review by falquaddoomi — APPROVED** [review]
Looks great, and I've verified that it's working on my end. Once we get phenogrid updated on npm, we can swap out that github reference for "^1.5.2".

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "lodash": "^4.17.15",
     "markdown-it": "^10.0.0",
     "markdown-it-anchor": "^5.2.5",
-    "phenogrid": "^1.5.1",
+    "phenogrid": "github:falquaddoomi/phenogrid#gulp4-upgrade",
```
Comment (kshefchek) [inline package.json:39]: should we just push this to npm?

Comment (kshefchek) [reply package.json:39]: testing it on netlify - it looks good to me

**Comment by vincerubinetti** [comment]
- update phenogrid package version and import properly

**Pull request merged**